src/com/gamecenter/TicTacToe.java

```diff
@@ -0,0 +1,91 @@
+package com.gamecenter;
+
+import javax.swing.*;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+
+public class TicTacToe extends Board{
+    TicTacToeButtonListener tacListener = new TicTacToeButtonListener();
+
+    int alternate = 0;
+    int count = 0;
+
+    private TicTacToe(){
+        setTicTacToe_Listener();
+    }
+    public  class TicTacToeButtonListener implements ActionListener {
+        public void actionPerformed(ActionEvent e) {
+            JButton buttonClicked = (JButton) e.getSource();
```

Comment:
conditional here checking if the button has an "X" or "O" then don't change and the player still has their turn to choose a spot.